bigquery_etl/util.py

```diff
@@ -100,18 +113,50 @@ def load_tables(
             source_format=table.source_format,
             write_disposition=bigquery.WriteDisposition.WRITE_TRUNCATE,
         )
+        has_bytes_fields = False
+
         if table.schema is None:
             # autodetect schema if not provided
             job_config.autodetect = True
         else:
             job_config.schema = table.schema
             # look for time_partitioning_field in provided schema
-            for field in job_config.schema:
+            for i, field in enumerate(job_config.schema):
+                if field.field_type == "BYTES":
+                    has_bytes_fields = True
                 if field.description == "time_partitioning_field":
                     job_config.time_partitioning = bigquery.TimePartitioning(
                         field=field.name
                     )
                     break  # stop because there can only be one time partitioning field
+
+        # If bytes fields, make them strings, and write this data to temp table
+        # Create the actual table as a read from that temp table
+        # NOTE: This doesn"t handle nested BYTES fields!!
+        if has_bytes_fields:
+            updated_schema = [
+                bigquery.schema.SchemaField(f.name, "STRING", f.mode)
+                if f.field_type == "BYTES"
+                else f
+                for f in job_config.schema
+            ]
+
+            job_config.schema = updated_schema
+            temp_table_name = "_" + table.name
+            destination = dataset.table(temp_table_name)
+            as_bytes = ",".join(
+                [
+                    f"FROM_HEX({f.name}) AS {f.name}"
+                    for f in table.schema
+                    if f.field_type == "BYTES"
+                ]
```

Comment:
If you supply hex encoded bytes in ndjson format for a bytes field bq load will already decode them from hex.

also, this would break if `f.name` is nested within an array, and probably also if it's nested within a struct.

Reply:
This could be useful, which hex representation? My only concern is that the input/output formats might be different in that case. Whatever we do we should ensure they are the same.

Reply:
er, sorry, not hex, base64

Reply:
and I believe it uses a base64 character set that matches python.

Reply:
That could work, and we could use `TO_BASE64` on the other end. I'll take a look

Reply:
Hmm, on second though a base-64 encoded byte string would be very confusing to look at, especially in tests...

Reply:
> That could work, and we could use `TO_BASE64` on the other end. I'll take a look

why would we need to use `TO_BASE64` on the other end?

Reply:
> Hmm, on second though a base-64 encoded byte string would be very confusing to look at, especially in tests...

base64 is also what the bigquery console and bq tool use to represent bytes, and what is used throughout ingestion-beam tests. I think we should stick with base64